waveform_editor/gui/selector/options_button_row.py

```diff
@@ -79,11 +80,7 @@ def __init__(
             on_click=self._on_rename_waveform_button_click,
             visible=has_waveforms,
         )
-        self.rename_waveform_panel = TextInputForm(
-            "",
-            is_visible=False,
-            on_click=self._rename_waveform,
-        )
+        self.rename_waveform_modal = RenameModal()
```

Comment:
Can we put this on the main component (like the ConfirmModal) instead of creating one instance per group?